### DISCUSSION TITLE
Show currently playing track from other clients

### DISCUSSION THREAD

**Discussion by voltagex** [navidrome/navidrome#3196] [Ideas]
![image](https://github.com/user-attachments/assets/f54d1184-a14c-4a18-acd5-3b69782562f5)

I'd expect to be able to see what I'm playing from play:sub here, or somewhere close to it.

At least, I'd like to quickly be able to see what the server is doing or whether there's any transcoding going on.

**Comment by deluan** [comment]
That was the plan, an activity panel similar to the one on Plex. And we even have an old, unfinished PR (probably closed by now) to add that. But now it does not make sense to add it to the current UI, as we are preparing to rewrite the UI.
